spec/controllers/chapters_controller_spec.rb

```diff
@@ -2,33 +2,33 @@
 
 describe ChaptersController do
   context "routing" do
-    let(:boston){ FactoryGirl.build_stubbed(:chapter) }
-
     it "routes /chapters/boston to chapters#show" do
-      {:get => "/chapters/boston"}.should
-        route_to({:controller => "chapters", :action => "show", :id => boston.id, :locale => "en"})
```

Comment:
These tests were not actually running -- Ruby interpreted line 9 as separate from line 8. This basically ran `{get: "/chapters/boston"}.should`, which passes because it's truthy, and then the `route_to` just returns an expectations object that's never eval'd.